### PR TITLE
Refactor using Javascript classes 

### PR DESCRIPTION
**Pull request by baqar-abbas**
Refactor the previous project using Javascript classes. Added main.js file and all javascript defined there. Added styling in styles.css and updated index.html file accordingly. 
This project has been completed in collaboration with my coding partner Fuad Moin.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
+/* eslint-disable linebreak-style */
```
Comment (mckent05) [inline main.js]: - Please remove all disabled linters, remember that linters help us ensure that we write code that is clean and easy to maintain. Please fix this 🙏

File: main.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        listItem.remove();
+      });
+
+      listItem.appendChild(removeButton);
+      list.appendChild(listItem);
+    });
+  }
+}
+
+Book.displayAll();
```
Comment (mckent05) [inline main.js]: - I suggest calling this method using an event listener, this would make your code look more readable. Please fix this 🙏

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @baqar-abbas, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 
- Your code works as expected, good job 👍 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.js (hunk 1 above)
Comment (baqar-abbas) [reply main.js]: Suggested changes incorporated and disable statement removed screen shot attached for your kind reference 
image

File: main.js (hunk 2 above)
Comment (baqar-abbas) [reply main.js]: Above function is also there in the event listener  kindly refer to the screen shot below.
image
This function call above is for page refresh now we have put this function call into window.onload event to display the data when page refreshes.
All suggested changes have been made and sending for a review again.

**Review by DammyShittu — APPROVED** [review]
Hi @baqar-abbas and @fuadmoin,

## APPROVED 🟢

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**